### PR TITLE
Update #608

### PR DESCRIPTION
**Pull request by J0rtIT**
Added important Note #608

### CODE REVIEW

**Comment by J0rtIT** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

File: microsoft-365/business/access-resources.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ Any Windows 10 device that is Azure Active Directory joined has access to all cl
 To learn more, see [Introduction to device management in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/device-management-introduction).
 The steps are also summarized in the following sections.
 
+> [!IMPORTANT]
+> This procedure is only for OAuth and NTLM and Kerberos is not supported.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/access-resources.md]:
```suggestion
> This procedure is only applicable to OAuth and NTLM. Kerberos is not supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review based on issue #608.  

Thank you.

**Review by Sirkkuw — APPROVED** [review]

**Pull request merged**